Add files via upload

Andreirad89 opened the pull request:
Код первой домашней работы.

Andreirad89 committed:
Add files via upload

Код первой домашней работы.

AleksandrGladkovIT commented on HomeWork1/HomeWorkApp.java:
```diff
@@ -0,0 +1,56 @@
+        if (value <= 0) {
+            System.out.println("Красный");
+        }
+        if (value > 100) {
```
if (value > 0 && value <= 100)

Andreirad89 committed:
Add files via upload

homework2

Andreirad89 commented:
вторая домашняя работа

Andreirad89 committed:
ДЗ №3

Andreirad89 committed:
ДЗ №4

Сделал битву. Обратил внимание что при прохождении первого врага его HP <=0 соответственно у оставшихся врагов уровень HP такой же. Добавил строку 171. Весь бой от 156 до 171(

Andreirad89 committed:
ДЗ №5

Сделана работа №5

Andreirad89 committed:
Delete HomeWorkApp.java

Andreirad89 committed:
Delete HomeWorkAppTwo.java

Andreirad89 committed:
Delete home4.java

Andreirad89 committed:
Delete homework3.java

Andreirad89 committed:
Delete Master.java

Andreirad89 committed:
Delete Worker.java
